CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to the Open Liberty Tools extension will be documented below.
 
+## 0.1.10
+- Honour Maven and Gradle wrapper settings
```

Comment:
Maybe "Honour Maven and Gradle wrapper settings when running dev mode." 

Also should probably document somewhere on the README what settings are avaliable and maybe which plugin they are from.